src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -150,12 +153,12 @@
       </_SupplementalTestData>
     </ItemGroup>
     <ItemGroup>
-      <SupplementalTestDataTestDir Include="@(_SupplementalTestData->'$(TestPath)/%(DestinationDir)%(Filename)%(Extension)')" />
```

Comment:
I must have missed part of that discussion because while I'm OK with leaving CopySupplementalData as an item group I still believe it should be copied into Content as part of the build so we don't need to special case it anywhere else in our targets.